server/helpers/ffmpeg-utils.ts

```diff
@@ -156,6 +169,104 @@ async function generateImageFromVideoFile (fromPath: string, folder: string, ima
   }
 }
 
+/**
+ * Generates images representing timecodes of the video, used for scrubbing preview
+ *
+ * @param fromPath source path
+ * @param folder destination folder
+ * @param imageName base image name
+ * @param size dimensions to which the images should be resized
+ * @param count images/video, always capped at 1 image/sec
+ */
+async function generateTimecodeThumbnailsFromVideoFile (
+  fromPath: string,
+  folder: string,
+  imageName: string,
+  size: {
+    width: number,
+    height: number
+  },
+  count = 50
+) {
+  // cap thumbnails at 1 image/sec max
+  let duration = await getDurationFromVideoFile(fromPath)
+  if (count > duration) count = duration
+
+  imageName = TimecodeThumbnailManifestModel.generateManifestName().replace(/\.[^.]+$/, extname(imageName))
```

Comment:
Please create a dedicated function to generate the image name, without a `replace`